mne/evoked.py

```diff
@@ -1436,6 +1436,77 @@ def _write_evokeds(fname, evoked, check=True):
         end_file(fid)
 
 
+@fill_doc
+def export_evokeds(fname, evoked, fmt='auto', device=None, history=None):
```

Comment:
If we force the `mff_*` entries to be provided as kwargs, it makes for nice future compat:
```suggestion
def export_evokeds(fname, evoked, fmt='auto', *, mff_device=None, mff_history=None):
```
because later we can add to the signature *before* the MFF-specific entries if there happens to be something common to all formats for example.